### DISCUSSION TITLE
Not receiving form submissions

### DISCUSSION THREAD

**Discussion by shagariboy** [emailjs-com/help#100] [Q&A]
Hi,
I got a problem. I am not receiving form submissions to the email "to email" in this form: [Booking form](https://github.com/shagariboy/interwest_travels/blob/main/BookingForms/zan_form.html). I have tested the service, template with another form (not in this website) and they work. This the repo: [website](https://github.com/shagariboy/interwest_travels). 

The account is under "nnamdiezeh0@gmail.com" with public key: teNl4WUP9Sj8LxBNR
<img width="950" alt="Screenshot 2024-04-11 173332" src="https://github.com/emailjs-com/help/assets/45914831/75e50f47-66ee-4f93-83f2-1f4e0b5c167f">

**Comment by xr0master** [comment]
Judging by the screenshot, you are sending emails to nnamdiezeh0@gmail.com

**Reply by shagariboy** [reply]
Yes, but I am not receiving anything in the mail.

**Reply by xr0master** [reply]
what does mean "not receiving anything"?

**Reply by shagariboy** [reply]
I mean when i submit form, i don't receive the form submissions in "nnamdiezeh0@gmail.com"
